Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM docker.io/alpine:edge as builder
```

Comment:
I'm a little concerned if we're using edge (= latest version) as the build dependencies, but base-alpine will depend on a fixed version (currently 3.7)

Reply:
I remember why I moved to edge: rust 1.22 is the latest on 3.7, edge has 1.24, which we need for actix.

Reply:
Had convo IRL: edge is updated to rust 1.26, alpine 3.8 will be released soon (with 1.26 hopefully) and we will move to that asap. For now, to avoid having version mismatches between the builder and running container, both are based on edge.